sdk/servicefabricmanagedclusters/Azure.ResourceManager.ServiceFabricManagedClusters/CHANGELOG.md

```diff
@@ -12,7 +12,13 @@
 
 ### Other Changes
 
-## 1.2.0 (2026-06-25)
+## 1.2.0 (2024-10-01)
```

Comment:
There is already a `1.2.0`, you need to specify a new version or update the existing version. Besides, for preview API, do you really want to release a production version?